codegenerator/integration_tests/src/main.rs

```diff
@@ -153,8 +153,8 @@ async fn all_hypersync_networks_are_included() {
 
 #[tokio::main]
 async fn main() {
-    all_hypersync_networks_are_included().await;
     run_all_init_combinations().await;
+    all_hypersync_networks_are_included().await;
```

Comment:
Changed the order, so we can at least know that the `run_all_init_combinations` passes when `all_hypersync_networks_are_included` fails.

Reply:
I was actually thinking we should rather put these as separate tests in a suite and then run the tests.

Reply:
Yeah, I'll do it next time it annoys me 😅